test/framework/toolchain.py

```diff
@@ -948,6 +948,44 @@ def test_nosuchtoolchain(self):
         tc = self.get_toolchain('intel', version='1970.01')
         self.assertErrorRegex(EasyBuildError, "No module found for toolchain", tc.prepare)
 
+    def test_mpi_cmd_prefix(self):
+        """Test mpi_exec_nranks function."""
+        self.modtool.prepend_module_path(self.test_prefix)
+
+        tc = self.get_toolchain('gompi', version='2018a')
+        tc.prepare()
+        self.assertEqual(tc.mpi_cmd_prefix(nr_ranks=2), "mpirun -n 2 ")
```

Comment:
we should strip off the trailing whitespace imho, to avoid duplicate spaces when using something like:

```python
sanity_check_commands = ["%(mpi_cmd_prefix)s example"]